django/core/management/commands/dumpdata.py

```diff
@@ -6,6 +8,40 @@
 from django.core.management.utils import parse_apps_and_model_labels
 from django.db import DEFAULT_DB_ALIAS, router
 
+try:
+    import bz2
+    has_bz2 = True
+except ImportError:
+    has_bz2 = False
+
+try:
+    import lzma
+    has_lzma = True
+except ImportError:
+    has_lzma = False
+
+
+def get_open_method(file_path):
```

Comment:
Should this be a method on the Command, so that it can be subclassed, and folks can add `NewHotnessFormatXWriter` without us having to handle that? 

* It could just be proxied to a method if we prefer the function. 
* _"No, it's fine"_ might also be the answer. 

Then:

* Should such a method be documented? 
* Is there any action in adjusting the empty `kwargs` dict?